examples/travel-lb.amp.html

```diff
@@ -0,0 +1,197 @@
+    content: '';
+    display: block;
+    width: 20px;
+    height: 20px;
+    background-image: url('data:image/svg+xml,<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12"><path fill="#fff" fill-rule="evenodd" d="M11.53,0.47 C11.237,0.177 10.762,0.177 10.469,0.47 L5.999,4.94 L1.529,0.47 C1.236,0.177 0.761,0.177 0.468,0.47 C0.175,0.763 0.175,1.238 0.468,1.531 L4.938,6.001 L0.468,10.471 C0.175,10.764 0.175,11.239 0.468,11.532 C0.761,11.825 1.236,11.825 1.529,11.532 L5.999,7.062 L10.469,11.532 C10.762,11.825 11.237,11.825 11.53,11.532 C11.823,11.239 11.823,10.764 11.53,10.471 L7.06,6.001 L11.53,1.531 C11.823,1.238 11.823,0.763 11.53,0.47 L11.53,0.47 Z"/></svg>');
+    background-repeat: no-repeat;
+    background-size: 100%;
+  }
+
+  .highlight-square {
+    content: '';
+    background-color: #20bf7c;
+    width: 5px;
+    height: 5px;
+    position: absolute;
+    bottom: 5px;
+    margin: auto;
+    left: 0;
+    right: 0;
+  }
+  </style>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async custom-element="amp-lightbox" src="https://cdn.ampproject.org/v0/amp-lightbox-0.1.js"></script>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <script async custom-element="amp-date-picker" src="https://cdn.ampproject.org/v0/amp-date-picker-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+<div class="search">
+  <p>Choose your travel dates</p>
+  <div class="fields">
+    <input on="tap:lb" placeholder="Departing" id="departing"/>
+    <input on="tap:lb" placeholder="Returning" id="returning"/>
+    <button on="tap:dp.clear">clear</button>
+  </div>
+</div>
+<amp-lightbox id="lb" layout="nodisplay">
+  <button class="lightbox-close" on="tap:lb.close" tabindex="0"></button>
+  <div class="date-picker-wrapper">
+    <amp-date-picker
+      class="date-picker"
+      id="dp"
+      type="range"
+      number-of-months="1"
+      src="travel.json"
+      format="MM/DD/YYYY"
+      week-day-format="dd"
+      reopen-picker-on-clear-date
+      on="
+        activate: lb;
```

Comment:
Question: What is the `activate: lb` doing? I assume the `activate` method for lightbox is open, but this thing is already in an opened lightbox, If I am missing and this is needed, let's call `lb.open` I am not a fan of the implicit activate method we run on actions with just the id.

Reply:
The idea here is the inputs are not in the lightbox, while the date picker calendar is inside the lightbox. The `activate` event allows the lightbox to open and render the date picker when the inputs are interacted with via the mouse or the keyboard down arrow.

Reply:
I am still missing how this works

I understand inputs opening the ligthtbox via `tap:lb.open`
I can't reason what the `activate:lb` code on `amp-date-picker` does. `lb` is already open when `amp-date-picker` activates, no?

Reply:
Activate is called when the user would be interacting with the date-picker ui. For example, when the user presses `DOWN_ARROW` with the input focused, `activate` will trigger and focus will be assigned to the date picker UI.

Reply:
I see now. so `focus` itself is not considered activation (hence the need to have `tap:lb` on the inputs?) if so, would it make sense for `focus` to be considered activation as well?